roles/openshift_logging/library/openshift_logging_facts.py

```diff
@@ -214,7 +222,10 @@ def facts_for_configmaps(self, namespace):
             name = item["metadata"]["name"]
             comp = self.comp(name)
             if comp is not None:
-                self.add_facts_for(comp, "configmaps", name, item["data"])
+                self.add_facts_for(comp, "configmaps", name, dict(item["data"]))
+                if comp not in ["fluentd"]:
```

Comment:
The throttle config is in yaml format

Reply:
will we need to update this if/when we provide the kibana config as a yaml formatted configmap?

Reply:
There may need to be a more elegant way to handle this then... I can't remember why I made a special case out of fluentd and I'm surprised there isn't an issue with the fact that  throttle config is in yaml format. It may just be because we're currently only pulling values out of the ES configmap...?